ganga/ganga/__init__.py

```diff
@@ -43,7 +43,7 @@ def ganga_license():
 except Exception as x:
     logger.critical('Ganga system plugins could not be loaded due to the following reason: %s', x)
     logger.exception(x)
-    raise GangaException(x), None, sys.exc_info()[2]
+    raise(GangaException(x), None, sys.exc_info()[2])
```

Comment:
This is probably more Pythonicly written in Python3 as:
```suggestion
    raise GangaException.with_traceback(sys.exc_info()[2])
```